test/BasicKeyPairTest.template.ts

```diff
@@ -175,7 +175,7 @@ export const BasicKeyPairTester = (keyPairClass: KeyPairClass, deterministicPriv
                 // 2^252 + 27742317777372353535851937790883648493, little endian
                 const Group_Order = [
                     0xed, 0xd3, 0xf5, 0x5c, 0x1a, 0x63, 0x12, 0x58, 0xd6, 0x9c, 0xf7, 0xa2, 0xde, 0xf9, 0xde, 0x14, 0x00, 0x00, 0x00, 0x00,
-                    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x10,
+                    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
```

Comment:
while this might fix the test, it also changes what is being tested, so this is probably not what you want to do